config/application.rb

```diff
@@ -31,8 +32,10 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.eager_load_paths << "#{config.root}/app"
+    config.eager_load_paths += Dir["#{config.root}/app/queries"]
+    config.eager_load_paths += Dir["#{config.root}/app/commands"]
+    config.eager_load_paths += Dir["#{config.root}/app/presenters"]
```

Comment:
Are these changes required to use classic mode?

Reply:
No they aren't so could comment them out I guess. Make it easier to add them back in when the work has been done.